book/src/languages.md

```diff
@@ -59,7 +63,7 @@ These configuration keys are available:
 | `diagnostic-severity` | Minimal severity of diagnostic for it to be displayed. (Allowed values: `Error`, `Warning`, `Info`, `Hint`) |
 | `comment-token`       | The token to use as a comment-token                           |
 | `indent`              | The indent to use. Has sub keys `unit` (the text inserted into the document when indenting; usually set to N spaces or `"\t"` for tabs) and `tab-width` (the number of spaces rendered for a tab) |
-| `language-server`     | The Language Server to run. See the Language Server configuration section below. |
+| `language-servers`    | The Language Servers used for this language. See below for more information in the section [Configuring Language Servers for a language](#configuring-language-servers-for-a-language)   |
```

Comment:
I find this a bit confusing before reading all the examples, because it can be two types but it wasn't written clearly. Like one can be `{ name = "efm-lsp-prettier", only-features = [ "format" ] }` and one can be `"typescript-language-server"` and `"typescript-language-server"` means `{ name = "typescript-language-server" }`, and the user have to go through two sections to understand this.

Reply:
I see (and I have already seen how this wasn't clear for a few users here). Any suggestions how this could be improved?